grafana/notifications.libsonnet

```diff
@@ -0,0 +1,13 @@
+{
+  new(name, type, uid, org_id, settings, is_default=false, send_reminders=true, frequency='1h', disable_resolve_message=false):: {
```

Comment:
Maybe split this function signature over multiple lines for readability?

Reply:
that's not hard.